docs/custom-image.md

```diff
@@ -0,0 +1,115 @@
+In this section we will learn how we can create a custom BitOps image with plugins and additional utilities. 
```

Comment:
this page should have an example Dockerfile for a custom image (i.e. `FROM bitops:2.0.0-base`)